data-model/fdo-types/0.3.0/digital-media-objects/schema/digital-entity.json

```diff
@@ -0,0 +1,195 @@
+        "Software",
+        "Sound",
+        "StillImage",
+        "Text"
+      ]
+    },
+    "ac:accessUri": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/ac/terms/accessURI",
+      "examples": [
+      ]
+    },
+    "dwc:institutionId": {
+      "type": "string",
+      "description": "ROR or Wikidata identifier, based on https://rs.tdwg.org/dwc/terms/institutionID",
+      "examples": [
+        "https://ror.org/015hz7p22"
+      ],
+      "$comment": "Add format for ROR or Wikidata ID"
+    },
+    "ods:institutionName": {
+      "type": "string",
+      "description": "Full museum name according to ROR or Wikidata",
+      "examples": [
+        "National Museum of Natural History"
+      ],
+      "$comment": "Not part of DWC or the GBIF UM"
+    },
+    "xmpRights:webStatement": {
+      "type": "string",
+      "description": "https://ns.adobe.com/xap/1.0/rights/WebStatement",
+      "$comment": "What is the difference with dcterms:license?"
+    },
+    "dcterms:format": {
+      "type": "string",
+      "description": "https://purl.org/dc/terms/format",
+      "$comment": "Create a enum for this?"
+    },
+    "dcterms:license": {
+      "type": "string",
+      "description": "https://purl.org/dc/terms/license",
+      "$comment": "Create a enum for this?"
+    },
+    "dcterms:description": {
+      "type": "string",
+      "description": "https://purl.org/dc/terms/description"
+    },
+    "dcterms:rights": {
+      "type": "string",
+      "description": "https://purl.org/dc/terms/rights",
+      "$comment": "What is the difference with dcterms:license?"
+    },
+    "ods:rightsUri": {
+      "type": "string",
+      "format": "uri",
+      "$comment": "What is the difference with dcterms:license? or dcterms:rights?. dcterms:rights is already recommended to be a URI"
+    },
+    "dcterms:accessRights": {
+      "type": "string",
+      "description": "https://purl.org/dc/terms/accessRights",
+      "$comment": "What is the difference with dcterms:license?"
+    },
+    "dcterms:rightsHolder": {
+      "type": "string",
+      "description": "https://purl.org/dc/terms/rightsHolder",
+      "examples": [
+      ]
+    },
+    "dcterms:source": {
+      "type": "string",
+      "description": "https://purl.org/dc/terms/source"
+    },
+    "ods:sourceUri": {
+      "type": "string",
+      "format": "uri",
+      "$comment": "It is already recommended to use URI in the dcterms:source"
+    },
+    "dcterms:creator": {
+      "type": "string",
+      "description": "https://purl.org/dc/elements/1.1/creator"
+    },
+    "dcterms:created": {
+      "type": "string",
+      "description": "https://purl.org/dc/terms/created"
+    },
+    "dcterms:modified": {
+      "type": "string",
+      "description": "https://purl.org/dc/terms/modified"
+    },
+    "ods:Assertions": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "ods:Assertion": {
+          "type": "object",
```

Comment:
you need to put the properties into a "properties' block (see [stackoverflow here](https://stackoverflow.com/questions/36757949/json-schema-definition-for-array-of-objects))

i recommend something like:

```
{
  "ods:assertions": {
    "type": "array",
    "items": {
      "type": "object",
      "properties": {
        "dwc:assertion": {
          "$ref": "https://schemas.dissco.tech/schemas/digitalobjects/0.3.0/digital-specimens/assertions.json"
        }
      }
    }
  }
}
```

Reply:
i think you'd have to make this change in a lot of places

Reply:
Yes I seems you are right, will correct it on all places